docs/data/routes/docs/nextjs/troubleshooting/en.md

```diff
@@ -0,0 +1,156 @@
+---
+name: troubleshooting
+routeTemplate: ./data/component-templates/article.yml
+title: Documentation
+---
+
+# Troubleshooting
+
+This section has advice for troubleshooting issues with JSS development
+
+## Server-side JavaScript errors
+
+If you encounter unexpected JavaScript errors during the `npm install` or `build` steps, especially errors that cannot be reproduced by other team members, check the versions of Node and npm being used. From terminal, run:
+```
+node -v
+npm -v
+```
+Note that JSS is tested using the Long Term Support (TLS) versions of Node. These are typically one major version behind the latest official Node version.
+
+Note that the Node/npm version used by your CI/Production environments may be different from what is used by your local environment. If a project is not pinned to a specific Node/npm version in `package.json`, it is common for deployment agents to build using either the newest version available or an environment-specific "default" version.
+
+If you need to switch Node version on your local environment for testing, 3rd party packages like [n](https://github.com/tj/n) or [nvm]() can help with this task.
+
+System Pre-requisites
+Running Node LTS version (use node -v to check)
+
+## App does not build or Home Page shows 404
```

Comment:
Since we switched to `fallback: 'blocking'`, you wouldn't see a 404 any longer (at least when running w/ Next server). Should we change the header here to something like "Missing or unexpected pages when using `GraphQLSitemapService`" ? And mention 404 in static export scenario just below?

Reply:
Reorganized the content a bit to address this